tests/foreman/api/test_contentmanagement.py

```diff
@@ -368,8 +368,8 @@ def test_positive_checksum_sync(self, capsule_configured):
             f'grep -o \'checksum type="sha256"\' {lce_repo_path}/{REPOMD_PATH}',
         )
 
-        assert result.return_code != 0
-        assert len(result.stdout) == 0
+        assert result.status != 0
+        assert not len(result.stdout)
```

Comment:
The existing code is perfect to check if length is 0 instead of checking no length.

Reply:
With len only able to return an integer, these are effectively the same assertion. really, this could have been simplified further with `assert not result.stdout`

Reply:
I've been converting `assert len(thing) > 0` to `assert len(thing)` throughout the framework as I see it, because as @JacobCallahan said, `len` either returns 0 or a positive integer, and can act directly as a boolean.